src/test/resources/org/apache/maven/model/jdom/it/DependenciesEtlIT_removeLastDependency_expected-pom.xml

```diff
@@ -21,4 +21,5 @@
   <artifactId>my-test-project</artifactId>
   <version>1.0-SNAPSHOT</version>
 
+
```

Comment:
Why?

Reply:
Because this is the correct remove result. There is one line left.